src/content/docs/en/reference/api-reference.mdx

```diff
@@ -1,855 +1,245 @@
 ---
-<h1>{title}</h1>
-<ul>
-    {orders.map(order => {
-        return <li>{/* do something with each order */}</li>
-    })}
-</ul>
+<link rel="canonical" href={canonicalURL} />
+<meta property="og:image" content={socialImageURL} />
 ```
 
-### `Astro.preferredLocale`
-
-<p>
-
-**Type:** `string | undefined`<br />
-<Since v="3.5.0" />
-</p>
-
-`Astro.preferredLocale` is a computed value that represents the preferred locale of the user. 
-
-It is computed by checking the configured locales in your `i18n.locales` array and locales supported by the users's browser via the header `Accept-Language`. This value is `undefined` if no such match exists.
-
-This property is only available when building for SSR (server-side rendering) and should not be used for static sites.
-
-### `Astro.preferredLocaleList`
-
-<p>
-
-**Type:** `string[] | undefined`<br />
-<Since v="3.5.0" />
-</p>
-
-`Astro.preferredLocaleList` represents the array of all locales that are both requested by the browser and supported by your website. This produces a list of all compatible languages between your site and your visitor. 
-
-If none of the browser's requested languages are found in your locales array, then the value is `[]`: you do not support any of your visitor's preferred locales.
-
-If the browser does not specify any preferred languages, then this value will be [`i18n.locales`](/en/reference/configuration-reference/#i18nlocales): all of your supported locales will be considered equally preferred by a visitor with no preferences. 
-
-This property is only available when building for SSR (server-side rendering) and should not be used for static sites.
-
-### `Astro.currentLocale`
+### `site`
 
 <p>
 
-**Type:** `string | undefined`<br />
-<Since v="3.5.6" />
-</p>
-
-The locale computed from the current URL, using the syntax specified in your `locales` configuration. If the URL does not contain a `/[locale]/` prefix, then the value will default to `i18n.defaultLocale`.
-
-### `Astro.getActionResult()`
-
-<p>
-**Type:** `(action: TAction) => ActionReturnType<TAction> | undefined`<br />
-<Since v="4.15.0" />
+**Type:** `URL | undefined`
 </p>
 
-`Astro.getActionResult()` is a function that returns the result of an [Action](/en/guides/actions/) submission. This accepts an action function as an argument (e.g. `actions.logout`) and returns a `data` or `error` object when a submission is received. Otherwise, it will return `undefined`.
-
-```astro title="src/pages/index.astro"
----
-import { actions } from 'astro:actions';
-
-const result = Astro.getActionResult(actions.logout);
----
+`site` returns a `URL` made from `site` in your Astro config. It returns `undefined` if you have not set a value for [`site`](/en/reference/configuration-reference/#site) in your Astro config.
 
-<form action={actions.logout}>
-  <button type="submit">Log out</button>
-</form>
-{result?.error && <p>Failed to log out. Please try again.</p>}
+```astro title="src/pages/index.astro" "Astro.site"
```

Comment:
Flagging that I had asked for an example here, and Reuben provided a placeholder one. Will this work, or do we need to change it?

Reply:
I think the example is fine!